src/features/user_profile/components/SettingsForm/common.ts

```diff
@@ -0,0 +1,70 @@
+import { i18n } from '~core/localization';
+import { configRepo } from '~core/config';
+
+export const LANGUAGES = ['en', 'es', 'ar', 'ko', 'id', 'de', 'uk'] as const;
```

Comment:
Unneeded export. This constant is only used inside this file.
It also applies to the function in this file. We should probably removed unused exports